resources/assets/less/bem/forum-topics-spacer.less

```diff
@@ -16,23 +16,45 @@
  *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-.forum-topics-pager {
+.forum-topics-spacer {
   display: flex;
   align-items: center;
-  margin: 0px 40px;
-  min-height: 4em; // don't look too silly if there's no pager.
+  margin: 30px 40px;
 
   @media @mobile {
     // so the pager doesn't get squashed.
     // some reordering magic will probably be required if a right side button gets added.
     flex-direction: column;
   }
 
+  &--pager {
+    margin-top: 0;
+    margin-bottom: 0;
+    min-height: 4em; // don't look too silly if there's no pager.
+  }
+
   &__buttons {
     min-width: 150px;
+
     @media @mobile {
       width: 100%;
     }
+
+    // needs to:
+    // - expand to fit text.
+    // - not affect pager layout.
+    // - be the same width in blocks with no pager.
+    // - not expand to fill up space.
+    // - have minimum width.
+    // won't work if text is way too long.
+    .forum-topics-spacer--pager & {
```

Comment:
or maybe just `[[----flex: 1----][flex:none][----flex: 1----]]`? Paired with min-width set in `btn-osu-big--forum-new-topic` instead.